_data/site.yaml

```diff
@@ -1,11 +1,11 @@
-title: cloud.gov
-email: support@cloud.gov
+title: Agency Name
+email: contact@example.gov
 description: >- # this means to ignore newlines
   Agency Name (EAC) Lorem ipsum dolor sit amet, consectetur adipiscing elit.
   Aenean et sapien a leo auctor scelerisque quis nec magna. Sed dictum ante a risus vehicula facilisis.
 
 # Twitter handle. Only the handle, not the URL.
-twitter: clouddotgov
+twitter: 18F
```

Comment:
Keep the cloud.gov handle.

Reply:
This is only showing up because we reset the branch to 11ty.  The PR to migrate data should clean this up just fine.

Reply:
Once you rebase the next PR in the sequence, it will do this same diff down the line in the PR's. By fixing them know, future rebases will not have this conflict. The site data diff in `https://github.com/cloud-gov/site/pull/17/files#diff-5baa7a616344814920a8388f6ed5cd65bc704743d1900f35d2f244f0cfa1fdcdR3-R22` will disappear and we won't need to review it.

Reply:
@drewbo specifically asked that we have reviews set up like this and I reviewed the following with him and with @sknep:

MERGE FIRST First commit - clone of the 11ty Pages template as starting point
MERGE SECOND Initial setup - required libraries and their configuration
Data migration
JavaScript Migration
Styles migration
Layouts migration and main menu setup (prerequiste to content migration)
PDF documents migration
Images migration
Home page migration, contact, pricing, resources, signup, and support packages migration
Docs migration
Be sure to copy python script
Knowledge base migration
News migration
Dynamic sidenav setup and configuration
Pages’ layouts and header setup
Pages’ pages migration: home, contact, features, federalist, pricing, success stories
Pages’ documentation migration (includes sidenav)
Pages’ knowledge base migration

In this way, we can have atomic reviews of tightly related functionality and/or content migration.  Where that got messy is some work had already been reviewed and merged, I can't modify "main" directly, and github does not allow merging in an orphan branch with disparate histories as a true reset.  I even tried creating a branch off main and reset that to a point in time commit and created a peer request off of that, but unfortunately, github will refuse changes made that way.

An alternative would be just blow away this repo altogether... or, for someone with permissions, blow away main on this repo and create another one that is completely blank.

Reply:
I appreciate you laying out the plan. Let's merge this and rebase the PRs.